spec/index.html

```diff
@@ -7410,6 +7410,127 @@ <h5>TZ</h5>
             </div>
           </section>
         </section>
+
+        <!-- QT -->
+        <section id="func-quoted-triples">
+          <h4>Functions on Quoted Triples</h4>
+          <section id="func-triple">
+            <h5>TRIPLE</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">quoted triple</span>  <span class="operator">TRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">subj</span>, <span class="type RDFterm">RDF term</span> <span class="name">pred</span>, <span class="type RDFterm">RDF term</span> <span
+class="name">obj</span>)
+            </pre>
+            <pre class="query nohighlight">
+                  <span class="operator">&lt;&lt;</span> subj pred obj <span class="operator">&gt;&gt;</span>
+            </pre>
+            <p>
+              If the 3-tuple (<code style="color:black">subj</code>, 
+              <code style="color:black">pred</code>, 
+              <code style="color:black">obj</code>)
+              is an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>
+              (that is, <code style="color:black">subj</code> is an
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a> or a 
+              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>,
+              <code style="color:black">pred</code> is an 
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              and <code style="color:black">obj</code> is an
+              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a> or a 
+              <a data-cite="RDF12-CONCEPTS#dfn-literal">literal</a>)
+              the function returns a quoted triple with these three elements.
+              Otherwise, the function raises an error.
```

Comment:
```suggestion
              (that is, <code style="color:black">subj</code> is an
              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a> or
              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>;
              <code style="color:black">pred</code> is an
              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>;
              and <code style="color:black">obj</code> is an
              <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>,
              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
              <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a> or
              <a data-cite="RDF12-CONCEPTS#dfn-literal">literal</a>)
              the function returns a quoted triple with these three elements;
              otherwise, the function raises an error.
```

Reply:
I prefer to have a separate, short sentence.

Reply:
OK. The article adjustments are still needed.